README.md

```diff
@@ -2,7 +2,7 @@
 
 ## Installation
 
-See https://doc.rust-lang.org/book/ch01-01-installation.html
+See https://rustup.rs
```

Comment:
You might still want to keep the old URL for some reason I guess. But `https://rustup.rs` will detect the platform and select the correct installer automatically.